### PR TITLE
Make contact

### PR DESCRIPTION
**Pull request by LethaboMot**
in this project i:

-Added HTML
-Added CSS
-Added images

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -229,8 +229,21 @@ <h3 class="codeused  "> Skills
         </div>
     </section>
 
+    <footer class="contact-menu">
+
+        <h2 class="background">Contact me</h2>
+        <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+        <form class="contact-form" action="https://formspree.io/f/}" method="post">
```
Comment (Olamarx) [inline index.html:236]: - [ ] This action content is not complete, kindly create an account with formspree and then use your personal link. With this, you would be able to get the message directly in your mailbox.

File: index.html [hunk 2]
```diff
@@ -229,8 +229,21 @@ <h3 class="codeused  "> Skills
...
+    <footer class="contact-menu">
+
+        <h2 class="background">Contact me</h2>
+        <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+        <form class="contact-form" action="https://formspree.io/f/}" method="post">
+            <input type="name" id="name" name="name" placeholder="Yeremias" size="30" required>
+            <input type="email" id="email" placeholder="natajayanj14@gmail.com" size="30" required>
+            <input type="text" id="message" name="message" placeholder="Write your message here" required minlength="4" maxlength="500" size="100">
+            <div>
+                <button type="button" class="bluebutton2">Get In touch</button>
```
Comment (Olamarx) [inline index.html:241]: - [ ] This type should be submit so that it can interact with the form tag.

**Review by Olamarx — CHANGES_REQUESTED** [review]
Hi @LethaboMot, 

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Linters passed ✔️ 
- Form tag ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -229,8 +229,21 @@ <h3 class="codeused  "> Skills
...
+        <h2 class="background">Contact me</h2>
+        <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+        <form class="contact-form" action="https://formspree.io/f/mwkzgqen" method="POST">
+            <input type="text" id="name" name="text" placeholder="Yeremias" size="30" required>
+            <input type="email" id="email" placeholder="natajayanj14@gmail.com" size="30" required>
+            <input type="message" id="message" name="message" placeholder="Write your message here" required minlength="4" maxlength="500" size="100">
+            <div>
+                <button type="submit" class="bluebutton2">Get In touch</button>
+            </div>
+        </form>
```
Comment (frank1738) [inline index.html:243]: - [OPTIONAL] Kindly position your `form section` correctly to match the original design

image

### Checkout the original design

image

**Review by frank1738 — APPROVED** [review]
Hi @LethaboMot , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

✅ Correct use of Github workflow
✅ No linter error
✅ Form is getting submitted correctly👍 

## Optional suggestions

- check the comment below

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**